### PR TITLE
Add lots of Socket related bindings

### PR DESCRIPTION
**Pull request by sdroege**
Depends on https://github.com/gtk-rs/glib/pull/290

### CODE REVIEW

**Comment by EPashkin** [comment]
Last commit "Add SocketClient" don't contains regen with removing *_async

**Comment by sdroege** [comment]
> Last commit "Add SocketClient" don't contains regen with removing *_async

Thanks, fixed

File: src/resolver.rs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+use InetAddress;
+use SrvTarget;
+
+pub trait ResolverExtManual {
+    fn lookup_by_address_async<'a, P: Into<Option<&'a Cancellable>>, Q: FnOnce(Result<String, Error>) + Send + 'static>(&self, address: &InetAddress, cancellable: P, callback: Q);
+
+    fn lookup_by_name_async<'a, P: Into<Option<&'a Cancellable>>, Q: FnOnce(Result<Vec<InetAddress>, Error>) + Send + 'static>(&self, hostname: &str, cancellable: P, callback: Q);
+
+    #[cfg(any(feature = "v2_34", feature = "dox"))]
+    fn lookup_records_async<'a, P: Into<Option<&'a Cancellable>>, Q: FnOnce(Result<Vec<glib::Variant>, Error>) + Send + 'static>(&self, rrname: &str, record_type: ResolverRecordType, cancellable: P, callback: Q);
```
Comment (EPashkin) [inline src/resolver.rs]: No type this function
```
#[cfg(any(feature = "v2_34", feature = "dox"))]
use ResolverRecordType;
```

Comment (EPashkin) [reply src/resolver.rs]: also `#[cfg(any(feature = "v2_34", feature = "dox"))]use glib;` needed or `glib::Variant` with fixing type

**Comment by EPashkin** [comment]
All manual code looks good and PR compiled for windows.
@sdroege Thanks.

File: src/resolver.rs (hunk 1 above)
Comment (sdroege) [reply src/resolver.rs]: Thanks, fixed

**Comment by sdroege** [comment]
@GuillaumeGomez Should be good to go then, after the GLib one is merged :)

**Comment by GuillaumeGomez** [comment]
I restart CIs.

**Comment by sdroege** [comment]
CI happy :)

**Comment by GuillaumeGomez** [comment]
Then so am I. :smiley:

**Pull request merged**